packages/address-book-controller/src/AddressBookController.ts

```diff
@@ -32,13 +37,15 @@ export interface ContactEntry {
  * @property chainId - Chain id identifies the current chain
  * @property memo - User's note about address
  * @property isEns - is the entry an ENS name
+ * @property addressType - is the type of this address
  */
 export interface AddressBookEntry {
   address: string;
   name: string;
   chainId: string;
   memo: string;
   isEns: boolean;
+  addressType: string;
```

Comment:
Could we update this to use the enum as the type? Why not be more expressive with the type signature if we can.

Reply:
Done